### PR TITLE
ItemsRepeaterScrollHost API Review

### PR DESCRIPTION
**Pull request by MikeHillberg**

### CODE REVIEW

File: active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+or disappear. This item is called the anchor element, so ItemsRepeater and ScrollViewer coordinate
+to preserve the location of the anchor element on the screen (that is, in the view port).
+
+This coordination between ItemsRepeater and ScrollViewer is enabled by ScrollViewer implementing
+[IScrollAnchorProvider](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.IScrollAnchorProvider).
+But ScrollViewer ships in WUX, and didn't implement that interface until RS5, and
+so anchoring doesn't work for an ItemsRepeater on an app running on RS4 or below.
+
+The solution for this is the new `ItemsRepeaterScrollHost` control in this spec.
+This wraps the ScrollViewer, and and do the coordination with the ItemsRepeater.
```
Comment (micahl) [inline active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md]: typo

File: active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:controls="using:Microsoft.UI.Xaml.Controls">
+
+    <controls:ItemsRepeaterScrollHost>
+        <ScrollViewer>
+            <controls:ItemsRepeater ItemsSource='{x:Bind PeopleCollection}' />
+        </ScrollViewer>
+    </controls:ItemsRepeaterScrollHost> 
+    
+</Page
```
Comment (MikeHillberg) [inline active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md]: Typo: missing ">"

File: active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md (hunk 1 above)
Comment (micahl) [reply active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md]: Specifically, the 'and and do'

File: active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+# API Details
+```C#
+[webhosthidden]
+[content_property("ScrollViewer")]
+runtimeclass ItemsRepeaterScrollHost : Windows.UI.Xaml.FrameworkElement
+{
+    ItemsRepeaterScrollHost();
+    
+    Windows.UI.Xaml.Controls.ScrollViewer ScrollViewer { get; set; };
```
Comment (ChrisGuzak) [inline active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md]: nit, don't need `{ get; set; }` as that is the default

File: active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+[HorizontalAnchorRatio](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.ScrollViewer.HorizontalAnchorRatio)
+
+[VerticalAnchorRatio](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.ScrollViewer.VerticalAnchorRatio)
+
+[CurrentAnchor](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.ScrollViewer.CurrentAnchor)
+
+# API Details
+```C#
+[webhosthidden]
+[content_property("ScrollViewer")]
```
Comment (MikeHillberg) [inline active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md]: [contentproperty] (no underscore)

**Comment by jevansaks** [comment]
Adding reference to this review from the tracking issue Microsoft/microsoft-ui-xaml#460.

File: active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md (hunk 2 above)
Comment (jevansaks) [reply active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md]:
```suggestion
</Page>
```

File: active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md (hunk 4 above)
Comment (jevansaks) [reply active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md]:
```suggestion
[contentproperty("ScrollViewer")]
```

File: active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md (hunk 3 above)
Comment (jevansaks) [reply active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md]:
```suggestion
    Windows.UI.Xaml.Controls.ScrollViewer ScrollViewer;
```

File: active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+runtimeclass ItemsRepeaterScrollHost : Windows.UI.Xaml.FrameworkElement
+{
+    ItemsRepeaterScrollHost();
+    
+    Windows.UI.Xaml.Controls.ScrollViewer ScrollViewer { get; set; };
+    
+    Windows.UI.Xaml.UIElement CurrentAnchor { get; }    
+    
+    Double HorizontalAnchorRatio { get; set; };
```
Comment (jevansaks) [inline active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md]:
```suggestion
    Double HorizontalAnchorRatio;
```

File: active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+    Double VerticalAnchorRatio { get; set; };
```
Comment (jevansaks) [inline active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md]:
```suggestion
    Double VerticalAnchorRatio;
```

File: active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md (hunk 1 above)
Comment (jevansaks) [reply active/ItemsRepeaterScrollHost/ItemsRepeaterScrollHost.md]:
```suggestion
This wraps the ScrollViewer, and does the coordination with the ItemsRepeater.
```

**Pull request merged**